composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "ac26209d38878a5f59c2b8c104acff69",
```

Comment:
Please revert the commit https://github.com/findologic/plugin-plentymarkets-ceres-api/pull/191/commits/0bd17604b824ce7217d0539f5f6e4e1dc8b73ce3, as in this commit, all packages were updated, which was probably not your plan :D.